app/src/main/res/layout/item_list.xml

```diff
@@ -20,7 +20,16 @@
         android:tintMode="src_in"
         tools:ignore="ContentDescription" />
 
+    <View
```

Comment:
This is just a regular framelayout, but I've Overrided onInterceptTouch so I can delegate any touches to the right views. In this case either the HorizontalScrollView or this FrameLayout.